roadmap/index.md

```diff
@@ -4,6 +4,38 @@ layout: default
 
 # Roadmap
 
+## 0.6.0 (August 2018)
+
+### Custom Resource Definitions support for Topic Operator
+
+The Topic Operator should have support for [Custom Resource Definitions (CRDs)](https://kubernetes.io/docs/concepts/api-extension/custom-resources/).
+
+### Add support for encryption to Kafka Connect
+
+Add support for encryption and authentication for Kafka Connect.
+
+### Add support for authentication and authorization
+
+In order to use Strimzi for production workloads, it has to be possible to secure the cluster. This should 
+include:
+
+* Authentication using TLS client authentication and SASL
```

Comment:
SASL is rather a broad thing: What we're actually intending to implement is SASL SCRAM-SHA